lib/agoneum/web/templates/layout/app.html.slim

```diff
@@ -0,0 +1,23 @@
+doctype html
```

Comment:
I have no preference. I can just as easily add something to `.vimrc` to work with `.slim`